src/components/search/Search.jsx

```diff
@@ -1,45 +1,32 @@
-import React, { useRef } from 'react';
-import { useDispatch } from 'react-redux';
-import { getMoviesList } from 'service/index.js';
-import Col from 'react-bootstrap/Col';
-import Form from 'react-bootstrap/Form';
-import Button from 'react-bootstrap/Button';
+import React, { useState } from 'react';
+import { Link } from "react-router-dom";
+import { Col, Form } from 'react-bootstrap';
 import styles from './styles.module.css'
 
 const Search = () => {
     const placeholderTxt = 'What do you want to watch?';
     const findMovieTxt = 'Find your movie';
     const searchTxt = 'Search';
-    const input = useRef();
-    const dispatch = useDispatch();
 
-    const handleSubmit = (e) => {
-        const options = { search: input.current.value, searchBy: 'title', limit: '' };
-        e.preventDefault();
-        getMoviesList(dispatch, options);
-    }
+    const [query, setQuery] = useState('');
 
     const handleChange = (e) => {
-        if (e.target.value.length == 0) {
-            getMoviesList(dispatch);
-        }
+        setQuery(e.target.value);
     }
 
     return (
-        <Form onSubmit={handleSubmit}>
+        <Form>
             <Form.Label className={styles.formLabel}>
                 {findMovieTxt}
             </Form.Label>
             <Form.Row>
                 <Col>
                     <Form.Group controlId="formBasicEmail">
-                        <Form.Control type="search" placeholder={placeholderTxt} ref={input} onChange={handleChange} />
+                        <Form.Control type="search" placeholder={placeholderTxt} onChange={handleChange} />
                     </Form.Group>
                 </Col>
                 <Col>
-                    <Button type="submit" type="submit">
-                        {searchTxt}
-                    </Button>
+                    <Link className="btn btn-primary" to={`/search/${query}`}>{searchTxt}</Link>
```

Comment:
One curious thing about submitting search query is that it's send only upon clicking on 'Search' button, but any user would first hit 'Enter' button, yet it won't work here. You are free to leave it for now but please search for solutions of it in future in the web. Because it's a very general case.